src/View.Uno.WinUI/Themes/MergedResourceDictionaries.xaml

```diff
@@ -0,0 +1 @@
+<ResourceDictionary xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" />
```

Comment:
in your csproj, seems like xamlmerge does not take the right source of data. 

This file should also be added in gitignore